tests/providers/amazon/aws/operators/test_s3_bucket.py

```diff
@@ -22,7 +22,7 @@
 from moto import mock_s3
 
 from airflow.providers.amazon.aws.hooks.s3 import S3Hook
-from airflow.providers.amazon.aws.operators.s3 import S3CreateBucketOperator, S3DeleteBucketOperator
+from airflow.providers.amazon.aws.operators.s3_bucket import S3CreateBucketOperator, S3DeleteBucketOperator
```

Comment:
Why are you modifying s3 files?
I think you have several unrelated changes of s3 in this PR.